json-rpc/src/runtime.rs

```diff
@@ -122,6 +124,10 @@ pub fn bootstrap(
                 referer: info.referer(),
                 user_agent: info.user_agent(),
                 elapsed: info.elapsed(),
+                forwarded: info
+                    .request_headers()
+                    .get(header::FORWARDED)
```

Comment:
What's the header name?

Reply:
forwarded

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded

Reply:
So we know that HAproxy sets this header with the actual IP of the remote?

Reply:
I'll get that part done.